finetune/model.py

```diff
@@ -73,6 +73,33 @@ def language_model_op(X, M, params, featurizer_state):
         lm_predict_op = sample_with_temperature(lm_logits, params.lm_temp)
         return lm_predict_op, language_model_state
 
+    def masked_language_model_op(X, M, mlm_weights, mlm_ids, mlm_positions, params, featurizer_state):
+        language_model_state = masked_language_model(
+            X=X,
+            M=M,
+            mlm_weights=mlm_weights,
+            mlm_ids=mlm_ids,
+            mlm_positions=mlm_positions,
+            config=params,
+            embed_weights=featurizer_state["embed_weights"],
+            hidden=featurizer_state["sequence_features"],
+        )
+
+        lm_logits = language_model_state["logits"]
+        lm_logit_mask = np.zeros(
+            [1, lm_logits.get_shape().as_list()[-1]], dtype=np.float32
+        )
+        lm_logit_mask[:, encoder.vocab_size :] = -np.inf
+
+        if "use_extra_toks" in params and not params.use_extra_toks:
```

Comment:
Is this stuff really necessary? Can we actually decode from the model? If not we should probably just remove this

Reply:
Yeah we don't need that. I still need to refactor this chunk.  Got distracted dealing with missing base model weights and vocab entries yesterday :/